packages/icons/src/Svg.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Box, system } from '@marigold/system';
+
+type SvgProps = {
+  size?: number;
+  fill?: string;
```

Comment:
This is part of the svg props anyway.

Reply:
I think it's just the fill attribute. Width and height would be also possible.

Reply:
I meant that this is not necessary to add here, since you're extending the `SVGAttributes` anyway.

Reply:
I think ist necessary for the `size` prop.
"Property 'size' does not exist on type 'SystemProps<SvgProps, "svg">'." - if I delete line 5.

Reply:
Yes, I am only talking about the fill property 🙂

Reply:
Oh sure, sorry :D